### PR TITLE
Add file header comments and code comments and instructions for use d…

### PR DESCRIPTION
**Pull request by roboxxltd**
…ocuments

**Summary:**

fix: Add file header comments and code comments
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

**The PR fulfills these requirements:**

- [x] Related documents have been updated
- [ ] Related tests have been updated

### CODE REVIEW

**Comment by roboxxltd** [comment]
Fix style issues of code and comments

**Review by fqjun — COMMENTED** [review]
检查并删除注释前多余的空格

File: module/armor/basic_armor.hpp [hunk 1]
```diff
@@ -42,57 +51,64 @@ struct Armor_Data {
...
   int armor_type_th;
   int big_armor_aspect_max;
 };
 
 struct Light_Config {
+  // 绘图和调试开关
   int light_draw;
   int light_edit;
-
+  // 灯条高宽比范围
```
[the remark this answers is not in the document]
Comment (roboxxltd) [reply module/armor/basic_armor.hpp:80]: 这个是什么

[the remark this answers is not in the document]
Comment (roboxxltd) [reply module/armor/basic_armor.hpp:80]: 这个位置前面有空格

File: base/wolfvision.cpp [hunk 2]
```diff
@@ -1,9 +1,17 @@
+/**
+ * @file wolfvision.cpp
+ * @author XX (2796393320@qq.com) WCJJJ (1767851382@qq.com)
```
Comment (fqjun) [inline base/wolfvision.cpp:3]: 统一格式为分行

**Review by fqjun — CHANGES_REQUESTED** [review]
1. 主函数的文件注释改为主函数
2. 文件头部的作者注释格式统一为分行

File: base/wolfvision.hpp [hunk 3]
```diff
@@ -1,3 +1,12 @@
+/**
+ * @file wolfvision.hpp
+ * @author XX (2796393320@qq.com) WCJJJ (1767851382@qq.com)
```
Comment (fqjun) [inline base/wolfvision.hpp:3]: 作者注释统一格式为换行

File: module/roi/abstract_roi.hpp [hunk 4]
```diff
@@ -10,15 +19,29 @@ class RoI {
...
+  /**
+   * @brief 截取 ROI 图像
+   *
+   * @param _input_img 需要截取的图像
+   * @param _rect      需要截取的位置(外接矩形)
+   * @return cv::Mat   返回截取后的图片
+   * @author RCX
+   */
+  cv::Mat cutRoIRect(const cv::Mat&   _input_img,
                      const cv::Rect& _rect) {
```
Comment (fqjun) [inline module/roi/abstract_roi.hpp:31]: 未对齐

**Review by fqjun — CHANGES_REQUESTED** [review]
1. 个别位置没对齐
2. 作者的注释需进行格式统一

File: module/roi/abstract_roi.hpp (hunk 4 above)
Comment (roboxxltd) [reply module/roi/abstract_roi.hpp]: 已经修复完成

**Review by fqjun — COMMENTED** [review]
允许合并

**Pull request merged**